microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -44,9 +46,11 @@ Do not use User-Agent Client Hints to:
 - Block *unsupported* browsers.
 - Restrict access to features on your site.
 
-### User Agent Client Hints HTTPS header
+For more information, navigate to the W3C Community Draft Report at [User-Agent Client Hints][https://wicg.github.io/ua-client-hints/].

```

Comment:
Same here, looks like we have to fix up this link

Reply:
Option 1: A regular link syntax is [link text](URL).  
Option 2: A "reference" link syntax is two lines:
[link text][reference string describing the path]
[reference string describing the path]: url "title of destination | site or type of site"
See examples in repo.

Reply:
Missed that the first time, fixed now, thanks!